website_sale_qty/static/tests/js/website_sale_qty.tour.js

```diff
@@ -0,0 +1,105 @@
+/**
+*    Copyright 2016 LasLabs Inc.
+*    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+**/
+
+odoo.define('website_sale.tour', function(require) {
+    'use strict';
+
+    var tour = require('web.Tour');
+    var _t = require('web.core')._t;
+
+    tour.register({
+        id: 'test_website_sale_qty',
+        name: _t('Test JS for price/quantity radio buttons on product page in website shop'),
+        path: '/shop/product/ipad-mini-9',
+        mode: 'test',
+        steps: [
+            {
+                title: 'Check that the radio corresponding to a quantity of 1 is selected on load',
+                onload: function() {
+                    var radio = $('.js-pqt-input[value="1"]');
+                    if (!radio.is(':checked')) {
```

Comment:
Actually, the `waitFor`/`waitNot` approach doesn't seem to work properly here. For example, the tour will still finish successfully even if this step includes a line like `waitFor: '.js-pqt-input[value="1"]:not(:checked)'`. Something similar happens with the last step in the tour, where `waitNot: '.js-pqt-input[value="5"]:checked'` doesn't generate an error. And I don't think there's any way to check the value **property** of an `<input>` as part of a jQuery selector, which would be necessary to replace the `if` statement in the step below.

In light of all these issues, I decided to just go with the `onload` callback throughout for consistency.